roxctl/generate/netpol/netpol.go

```diff
@@ -21,33 +29,60 @@ type generateNetpolCommand struct {
 func Command(cliEnvironment environment.Environment) *cobra.Command {
 	generateNetpolCmd := &generateNetpolCommand{env: cliEnvironment}
 	c := &cobra.Command{
-		Use: "netpol <folder-path>",
+		Use:  "netpol <folder-path>",
+		Args: cobra.ExactArgs(1),
 		RunE: func(c *cobra.Command, args []string) error {
-			if err := validate(args); err != nil {
+			if err := generateNetpolCmd.construct(args, c); err != nil {
 				return err
 			}
-			if err := generateNetpolCmd.construct(args); err != nil {
+			if err := generateNetpolCmd.validate(); err != nil {
 				return err
 			}
 			return generateNetpolCmd.generateNetpol()
 		},
 	}
-
-	c.Flags().BoolVar(&generateNetpolCmd.offline, "offline", false, "whether to connect to a central instace for additional information")
+	c.Flags().BoolVar(&generateNetpolCmd.removeOutputPath, "remove", false, "remove the output path if it already exists")
+	c.Flags().StringVarP(&generateNetpolCmd.outputFolderPath, "output-dir", "d", "", "save generated policies into target folder - 1 file per policy")
```

Comment:
Numbers between 0-10 should be spelled out. Maybe use "single", or "one"?
```suggestion
	c.Flags().StringVarP(&generateNetpolCmd.outputFolderPath, "output-dir", "d", "", "save generated policies into target folder - single file per policy")
```